pkg/authorize/ssl/ssl_test.go

```diff
@@ -53,10 +79,29 @@ func TestClientCertInfoAsHeaders(t *testing.T) {
 			},
 			expect: http.StatusForbidden,
 		},
+		{
+			name: "Test missing CN returns 403",
```

Comment:
We're not testing the missing CN here. We are being rejected because the secret is not set. I think that for these tests, we should hace a base request that works from which we are removing parts to make it selectively fail.